### ISSUE TITLE
Editor crashes upon opening ocean_trackview level from AutomatedTesting project

### ISSUE THREAD

**Issue by LB-KacperKapusta** [o3de/o3de#11904]
**Describe the bug**
Opening the ocean_trackview level from AutomatedTesting project causes Editor to crash.

Please note that the same issue occurs for both Windows and Linux OS.

For further information please refer to the attached video and crash logs.

**Call Stack Trace**
```
25) 00007FFC06A90057 (KERNELBASE) : UnhandledExceptionFilter
24) 00007FFBF313D6D8 (InProcessClient64) : (function-name not available)
23) 00007FFBF313D321 (InProcessClient64) : (function-name not available)
22) 00007FFC091953B0 (ntdll) : memset
21) 00007FFC0917C766 (ntdll) : _C_specific_handler
20) 00007FFC0919229F (ntdll) : _chkstk
19) 00007FFC09141454 (ntdll) : RtlRaiseException
18) 00007FFC09190DCE (ntdll) : KiUserExceptionDispatcher
17) C:\o3de\Code\Editor\TrackView\TrackViewSequence.cpp (86) : CTrackViewSequence::Load
16) C:\o3de\Gems\Maestro\Code\Source\Components\EditorSequenceComponent.cpp (158) : Maestro::EditorSequenceComponent::Init
15) C:\o3de\Code\Framework\AzCore\AzCore\Component\Entity.cpp (176) : AZ::Entity::Init
14) C:\o3de\Code\Framework\AzToolsFramework\AzToolsFramework\Entity\EditorEntityContextComponent.cpp (719) : AzToolsFramework::EditorEntityContextComponent::SetupEditorEntities
13) C:\o3de\Code\Framework\AzFramework\AzFramework\Entity\EntityContext.cpp (178) : AzFramework::EntityContext::HandleEntitiesAdded
12) C:\o3de\Code\Framework\AzCore\AzCore\EBus\Internal\BusContainer.h (1541) : AZ::Internal::EBusContainer<AzToolsFramework::EditorEntityContextRequests,AzToolsFramework::EditorEntityContextRequests,0,0>::Dispatcher<AZ::EBus<AzToolsFramework::EditorEntityCon
11) C:\o3de\Code\Framework\AzToolsFramework\AzToolsFramework\Prefab\Instance\InstanceUpdateExecutor.cpp (265) : AzToolsFramework::Prefab::InstanceUpdateExecutor::UpdateTemplateInstancesInQueue
10) C:\o3de\Code\Framework\AzCore\AzCore\EBus\Internal\BusContainer.h (1360) : AZ::Internal::EBusContainer<AZ::SystemTickEvents,AZ::SystemTickEvents,0,1>::Dispatcher<AZ::EBus<AZ::SystemTickEvents,AZ::SystemTickEvents> >::Broadcast<void (__cdecl AZ::SystemTic
 9) C:\o3de\Code\Framework\AzCore\AzCore\Component\ComponentApplication.cpp (1396) : AZ::ComponentApplication::TickSystem
 8) C:\o3de\Code\Editor\CryEdit.cpp (2345) : CCryEditApp::IdleProcessing
 7) C:\o3de\Code\Editor\CryEdit.cpp (2255) : CCryEditApp::OnIdle
 6) C:\o3de\Code\Editor\Core\QtEditorApplication.cpp (312) : Editor::EditorQtApplication::maybeProcessIdle
 5) 00007FFBB2B1F72A (Qt5Core) : QTimer::timeout
 4) 00007FFBB2B182DD (Qt5Core) : QObject::event
 3) 00007FFBB35A49A1 (Qt5Widgets) : QApplicationPrivate::notify_helper
 2) 00007FFBB35A3A23 (Qt5Widgets) : QApplication::notify
 1) 00007FFBB2AF17CB (Qt5Core) : QCoreApplication::notifyInternal2
```

**Steps to reproduce**
1. Launch Editor for AutomatedTesting project.
2. Open the ocean_trackview.prefab level.

**Expected behavior**
Level opens without issues.

**Actual behavior**
Editor crashes upon opening the ocean_trackview level.

**Video**

https://user-images.githubusercontent.com/86953659/189646088-59d6fb44-059b-4a21-84ea-7cf1cadb62b5.mp4

**Found in Branch**
Stabilization/2210 (c674d04)
Development (09185a6)

**Desktop/Device:**
 - Device: PC
 - OS: Windows
 - Version: 10
 - CPU: AMD Ryzen 5 3600
 - GPU: NVIDIA GeForce RTX 2060 SUPER
 - Memory: 16GB

**Additional context**
[error.zip](https://github.com/o3de/o3de/files/9547660/error.zip) (error.dmp)
[error.log](https://github.com/o3de/o3de/files/9547661/error.log)
[Editor.log](https://github.com/o3de/o3de/files/9547662/Editor.log)

**Comment by mbalfour-amzn** [comment]
The ocean_component and ocean_trackview levels just need to get deleted, we don't have an Ocean component in O3DE.

**Comment by LB-TomaszSerwanski** [comment]
The issue verified as Fixed in Stabilization/2210 (b83f535), however it still occurs in Development (134fe32).
